src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,37 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state:
+  {
+    categoryList: [],
+    errorMessage: '',
+  },
+  mutations:
+  {
```

Comment:
[nits]オブジェクトの改行位置は下記のような形が良いかなと思います！

```suggestion
  state: {
    categoryList: [],
    errorMessage: '',
  },
  mutations: {
```

Reply:
修正しました。